gulpfile.js

```diff
@@ -902,7 +902,11 @@ function enableLocalTesting(targetFile) {
 function performBuild(watch) {
   process.env.NODE_ENV = 'development';
   printNobuildHelp();
-  printConfigHelp(watch ? 'gulp watch' : 'gulp build');
+  let configHelpMessage = watch ? 'gulp watch' : 'gulp build';
```

Comment:
nit: This is a command, not the config help message. Since you've gotten this far, rename the string to `buildCommand` :)